libgsf.yaml

```diff
@@ -43,6 +43,12 @@ pipeline:
       expected-commit: ea9d8cd1369661bf62d0476474700a9e0887c812
       tag: LIBGSF_${{vars.mangled-package-version}}
 
+  - runs: |
+      git fetch origin master:master
+      # Fix failure to build from source against new libxml2, without
+      # nanohttp support. Ignore merge conflict in NEWS.
+      git cherry-pick -x 5d4bb55095d3d6ef793c1908a88504183e28644c || true
```

Comment:
Would we be better carrying this as a custom patch? otherwise when upstream do cut a new release with the fix, we're ignoring any errors so it may be harder to debug later failures